### DISCUSSION TITLE
SDK 1.7.0 -> non-existent fields (BT-148, BT-1481, BT-1482, BT-1483)

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#532] [Q&A] [answered]
Hi Labs,

Today we received a surprising error message in our system:
"If Received Tenders Micro (BT-1481) value is different from 'EMPTY' and Received Tenders Small (BT-1482) value is different from 'EMPTY' and Recevied Tenders Medium (BT-1483) value is different from 'EMPTY', then Received Tenders SME (BT-148) value must be equal to the sum of (Received Tenders Micro (BT-1481) values and Received Tenders Small (BT-1482) values and Recevied Tenders Medium (BT-1483) values)."
Error code: BR-BT-13713-0102"

I checked the CELEX, fields,json, ted doc website, but I couldn't find any of the above fields.
These fields maybe the codes of BT-760 Received Submissions Type.

Our system indicates errors in a field according to the schematron rules. This error message causes a problem because there is no such field in the system.

I don't understand these new fields. 
Is this just a typo or will these be new fields?

**Comment by YvesJo** [comment]
Hi,
This rule statement is based on an earlier definition of Received Submissions associated Business terms. Latest Business Terms to consider are BT-759 and BT-760 with the appropriate code. Statement will be updated in a next release.

**Comment by AnnaKSZ** [comment]
Thanks your reply

**Comment by AnnaKSZ** [comment]
Hi Lab,

>Statement will be updated in a next release.

To my greatest regret, the above important message has not been updated in SDK 1.8.0.
Which is the "next release" when do you update it?

Regards
Anna

**Reply by YvesJo** [reply, marked as answer]
Hi Anna,
Scopes for SDKs 1.8 and 1.9 were already frozen and the update is expected for SDK 1.10.
The new statement will be of the kind:

> The total number of tenders received from SMEs (i.e. Received Submission Count - BT-759-LotResult, for which Received Submission Type - BT-760-LotResult is set to 't-sme') must equal the sum of:
> 
> - the total number of tenders received from medium enterprises (i.e. Received Submission Count - BT-759-LotResult, for which Received Submission Type - BT-760-LotResult is set to 't-med'),
> - the total number of tenders received from small enterprises (i.e. Received Submission Count - BT-759-LotResult, for which Received Submission Type - BT-760-LotResult is set to 't-small'), and
> - the total number of tenders received from micro enterprises (i.e. Received Submission Count - BT-759-LotResult, for which Received Submission Type - BT-760-LotResult is set to 't-micro').

**Comment by AnnaKSZ** [comment]
Thank you for your reply and support.
